media_ui/src/main/res/drawable/language_spinner_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" >
+
+    <solid android:color="@color/black60alpha" />
+
+    <corners
+        android:bottomLeftRadius="24dp"
```

Comment:
The main difference in appearance here seems to be these 24dp corner radii. Have you checked how this new background looks with different text scaling?

Reply:
Yes, I tested it after your comments. Layout background is fine but I need to add some margin between the language text and caret sign. I will update it. Nich catch @Cliabhach. Thanks.

Reply:
Neat.

Reply:
....if that solution will take too long we can merge it in as-is. Just need to add a TODO and rebase on `dev`.

Reply:
Updated.

Reply:
All right! I've tried it out locally and it looks ok there too.